relay-server/src/actors/envelopes.rs

```diff
@@ -222,11 +223,7 @@ impl EnvelopeManager {
     ) -> Result<Self, ServerError> {
         // Enter the tokio runtime so we can start spawning tasks from the outside.
         #[cfg(feature = "processing")]
-        let runtime = tokio::runtime::Builder::new_multi_thread()
-            .worker_threads(1)
-            .enable_all()
-            .build()
-            .unwrap();
+        let runtime = utils::construct_runtime();
```

Comment:
IIUC this now creates a runtime which can send messages to actix, but it doesn't really need that.  I'd be fine with leaving this as calling the tokio builder directly, I don't mind a bit of being explicit as there's no tricky stuff going on like setting up the actix system that needs to be abstracted and documented.  (I appreciate @jan-auer may have desired to hide the tokio builder, I'll politely disagree 😉 )

Reply:
There are a few reasons for keeping this abstraction:

- There will be plenty of runtimes constructed for all the services once they are migrated. This is a single line and makes it easier to read, writing a 5+ line block every time will just clutter our constructor functions.
- It gives us a single place to hook central concerns or enforce patterns. For example, in a follow-up I will require to set explicit names for every runtime, so this can simply become a required parameter of that function.
- Initializing the actix system for every runtime is actually desired here, since that removes any risk of accidentally forgetting it. Also, this means there can be a single _TODO_ in the code base to remove it after the migration is done.

We will need to pass a flag or count to control whether this should be a single- or multi-threaded runtime, however.